3648/2/commands/start.rb

```diff
@@ -0,0 +1,45 @@
+require 'redis'
+require_relative 'help_methods'
+
+module StartCommand
+  include HelpMethods
+  def start!(*)
+    if registred?
+      respond_with :message, text: "Вы уже вошли под: #{session[:number]}"
+    else
+      save_context :register_from_message
+      respond_with :message, text: 'Введите ваш номер?'
+    end
+  end
+
+  def register_from_message(*words)
+    respond_with :message, text: register_session(words)
+  end
+
+  private
+
+  def register_session(number = nil, *)
```

Comment:
If you set default param it should help your method. In your case if you don't pass any param to that method call it will send an error `NoMethodError: undefined method first for nil:NilClass`